core/templates/dev/head/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -367,6 +368,13 @@ oppia.factory('ExplorationStatesService', [
         saveStateProperty(
           stateName, 'recorded_voiceovers', newRecordedVoiceovers);
       },
+      getSolicitAnswerDetails: function(stateName) {
```

Comment:
Have you used this method anywhere? I don't see any place where you are calling `StateSolicitAnswerDetailsService.init`, is it not required?

Reply:
Ignore the 2nd part of this comment (As I can see where and how you have initialized the service.)

Reply:
Btw its not being used anywhere, but yeah I have added it similar to the other properties.